<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>499</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>499</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SendingData">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>–û—Ç–ø—Ä–∞–≤–∏—Ç—å</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>201</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>501</width>
      <height>371</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>üìé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>üóë</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPortButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>–ü–æ—Ä—Ç —Å–æ–±–µ—Å–µ–¥–Ω–∏–∫–∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="ourPortLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>–°–æ–±—Å—Ç–≤–µ–Ω–Ω—ã–π</string>
    </property>
   </widget>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>–ö–æ–º–ø—å—é—Ç–µ—Ä 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
